src/content/peerconnection/constraints/js/main.js

```diff
@@ -290,10 +290,9 @@ setInterval(function() {
 // might be named toString?
 function dumpStats(results) {
   var statsString = '';
-  Object.keys(results).forEach(function(key, index) {
-    var res = results[key];
+  results.forEach(function(res) {
     statsString += '<h3>Report ';
-    statsString += index;
+    statsString += res.type;
```

Comment:
I find it a little confusing that we are adding the type to statsString twice. Maybe add a comment to why?